doxia-modules/doxia-module-markdown/src/test/resources/fenced-code-block.md

```diff
@@ -0,0 +1,5 @@
+Below code is Java:
+
+```java
+System.out.println(helloWorld);
```

Comment:
Quotes around literal missing

Reply:
Nah, it's on purpose. This is just a test resource that is translated as Sink events. But the presence of quotes around `helloWorld` breaks the *text* Sink event in several fragment, which is annoying for my test.

But we can consider *helloWorld* is a *String* variable defined somewhere else.

Again, this is just an example for the unit tests. We don't need valid Java code here.